browser/brave_tab_helpers.cc

```diff
@@ -67,6 +78,21 @@ void AttachTabHelpers(content::WebContents* web_contents) {
   BraveDrmTabHelper::CreateForWebContents(web_contents);
 #endif
 
+#if BUILDFLAG(ENABLE_BRAVE_WAYBACK_MACHINE)
+  auto* context = web_contents->GetBrowserContext();
+  if (!base::CommandLine::ForCurrentProcess()->HasSwitch(
```

Comment:
can we encapsulate this in brave wayback machine with `IsWaybackMachineEnabled` or similar?

Reply:
The purpose of this check and `IsWaybackMachineEnabled()` are different.
`IsWaybackMachineEnabled()` is for handling runtime wayback option change.
It is used inside the tab helper because option could be changed after attaching to tab helper.
So, existing WebContents can use wayback machine.

However, this switch is for not attaching tab helpers at all and below tor profile checking are for only attaching WaybackTabHelper to non tor-profile webcontents.